packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.base.tsx

```diff
@@ -80,10 +85,11 @@ export class ChoiceGroupBase extends BaseComponent<IChoiceGroupProps, IChoiceGro
           )}
           <div className={classNames.flexContainer}>
             {options!.map((option: IChoiceGroupOption) => {
-              const innerOptionProps: IChoiceGroupOptionProps = {
```

Comment:
Is there any way to keep this typing without getting an error when using `data-is-focusable` attribute? Every usage I can find in OUFR has no typing.